### ISSUE TITLE
deploy on RunPod serverless 

### ISSUE THREAD

**Issue by Josukje** [allburov/whatsapp-http-api#239]
is there a way to deploy on runpod, cuz I wanna use it for production.

**Comment by allburov** [comment]
Hi! it can be deployed on any environment with docker support.

If you want to save sessions between restart - you need to have a file volume outside of container to run the command below:
- https://docs.docker.com/storage/volumes/
- https://waha.devlike.pro/docs/how-to/sessions/
```bash
docker run -d -v `pwd`/.sessions:/app/.sessions -p 3000:3000/tcp --name whatsapp-http-api devlikeapro/whatsapp-http-api-plus
```

We're working on remote storage for sessions yet https://github.com/devlikeapro/whatsapp-http-api/issues/41

**Comment by devlikepro** [comment]
Hi! 
We know that it's been more then a year since the request was requested, but we're happy to announce MongoDB remote storage for authentication data is available in `2023.4.2` version!

This feature is now available as part of the [WAHA Plus](https://waha.devlike.pro/docs/how-to/plus-version/) offering.

We have documented the feature and its usage in our documentation, available at [Session Storage](https://waha.devlike.pro/docs/how-to/storages/). 

----

To get started with the MongoDB storage, you can follow these simple steps:

First, start the MongoDB server by running the following command:
```bash
docker run -d -p 27017:27017 --name mongodb mongo
```

Next, initiate the WAHA with the `WHATSAPP_SESSIONS_MONGO_URL` environment variable, and note the use of the `--network host` option as well:
```bash
docker run -e WHATSAPP_SESSIONS_MONGO_URL=mongodb://localhost:27017/ --network host devlikeapro/whatsapp-http-api-plus
```

Following these steps, you will have successfully configured the MongoDB storage for WAHA. All session authentication data will now be stored in the MongoDB database.


For those of you eager to test this new feature, we would like to stress the importance of thoroughly testing MongoDB storage before deploying it in any production-related infrastructure. 


[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
